app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<div>
+  <!-- Add new Post -->
+  <div class="mt-3 d-flex align-items-center flex-column">
+    <h1>Add new Post</h1>
+    <%= form_with model: @post, url: user_posts_path, method: :post, class:
+    'form' do |f|%> <%= f.text_field :title, class:"form-control" %>
+    <br />
+    <%= f.text_area :text, class:"form-control" %>
+    <br />
+    <%= f.submit "Post", class:"btn btn-primary" %> <% end %>
+  </div>
+</div>
```

Comment:
- [ ] your post is not creating, kindly fix this
![Screenshot from 2022-12-08 12-32-57](https://user-images.githubusercontent.com/66097371/206437036-90860873-0cca-48a9-af8a-3b8a7a036e43.png)